src/client/app/containers/import.ts

```diff
@@ -2,10 +2,14 @@ import { Component, ChangeDetectionStrategy } from '@angular/core';
 import * as fromRoot from '../reducers';
 import { Store } from '@ngrx/store';
 import { UI, Snippet, GitHub } from '../actions';
-import { AI, storage } from '../helpers';
+import { UIEffects } from '../effects/ui';
+import { environment, AI, storage } from '../helpers';
+import { Request, ResponseTypes } from '../services';
 import { Strings } from '../strings';
 import { isEmpty } from 'lodash';
 
+const viewUrlSettings = 'VIEW_URL_SETTINGS';
+
 //strings are specified inline (not imported) for performance reasons
```

Comment:
Not sure why this comment is there, and couldn't see in the file's history who added it; removed.